City-of-Pori/pori-kada
# PORI-123 Removed features extra boxes from configuration feature.

These are the footer information blocks that would get overriden when reverting features if they are in the feature.

To test:
1. Edit block 'Footer information FI', 'Footer information EN' and 'Footer information SV' on the page /admin/structure/block
2. drush fra -y && drush cc all
3. Make sure that the changes that you did to footer information blocks were not reverted.
